frontend/src/locales/en-US.json

```diff
@@ -517,7 +517,18 @@
     "upload-sql": "Upload SQL",
     "override-current-statement": "Override current SQL statement",
     "upload-sql-file-max-size-exceeded": "Max file size ({size}) exceeded.",
-    "waiting-earliest-allowed-time": "Will run after {time}"
+    "waiting-earliest-allowed-time": "Will run after {time}",
+    "batch-transition": {
+      "not-allowed-tips": "Some of selected issues cannot be {operation}",
+      "resolve": "Resolve",
+      "resolved": "resolved",
+      "cancel": "Cancel",
+      "cancelled": "cancelled",
+      "reopen": "Reopen",
+      "reopened": "reopened",
+      "action-n-issues": "{action} {n} issue | {action} {n} issues",
+      "successfully-updated-n-issues": "Successfully updated {n} issue | Successfully updated {n} issues"
```

Comment:
duplication?

Reply:
`action-n-issues` is the title of the modal dialog. e.g., "Resolve 3 issues"
`successfully-updated-n-issues` is the content of the notification after updating.

Reply:
Whey we have "Successfully updated {n} issue | Successfully updated {n} issues" instead of just "Successfully updated {n} issue"

Reply:
The "one sentence | another sentence" format is how we handle plurality in vue-i18n.